tests/bundle/Functional/ImageVariationTest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformRestBundle\Tests\Functional;
+
+use EzSystems\EzPlatformRestBundle\Tests\Functional\TestCase as RESTFunctionalTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ImageVariationTest extends RESTFunctionalTestCase
+{
+    public function testCreateContent(): string
+    {
+        $string = $this->addTestSuffix(__FUNCTION__);
+        $fileName = basename('1px.png');
```

Comment:
```suggestion
        $fileName = '1px.png';
```
should give you the same result, right?